install/install.go

```diff
@@ -988,6 +991,7 @@ type k8sInstallerImplementation interface {
 	GetNamespace(ctx context.Context, namespace string, options metav1.GetOptions) (*corev1.Namespace, error)
 	ListPods(ctx context.Context, namespace string, options metav1.ListOptions) (*corev1.PodList, error)
 	DeletePod(ctx context.Context, namespace, name string, options metav1.DeleteOptions) error
```

Comment:
Is `DeletePod` not needed anymore after this patch?

Reply:
I didn't check if `DeletePod` was referenced in other parts of the code. If it's not, it could make sense to remove that.

Reply:
Yeah it doesn't look like it's used anywhere else so we can remove it.

Reply:
just pushed a new commit with the function deleted